G3D10/G3D-app.lib/include/G3D-app/DefaultRenderer.h

```diff
@@ -201,6 +285,156 @@ class DefaultRenderer : public Renderer {
 
 public:
 
+    // Named constants for the number of frames to initialize.
+    static const int CAMERA_TRACK = 2;
+    static const int SCENE_INIT = 5;
+
+    /** For creating the diffuse irradiance probe volume(s).
+        Volumes at varying grid resolutions store irradiance (RGB10A2) and
+        mean distance/squared-distance (RG16F). When enabled, these volumes are
+        updated using raytracing and queried during shading (for both raytracing
+        and rasterization) for duffise global illumination. Details in:
+
```

Comment:
This comment is probably fine but in case we want to describe the side effects of this function, I notice the following happens:

* Clear and append new `DDGIVolumes` to `DefaultRenderer::m_ddgiVolumeArray`.
* `DefaultRenderer::m_ddgiVolumeArray` gets copied into the `environment.ddgiVolumeArray`.
* Clear and set to false entries in `DefaultRenderer::m_showProbeLocations`.